### ISSUE TITLE
Having multiple tflite models deployed on same application

### ISSUE THREAD

**Issue by nitish11** [asus4/tf-lite-unity-sample#320]
**Is your feature request related to a problem? Please describe.**
How can we implement two tflite models deployed in the same application? For example, let's say I want to detect objects using SSD model and mobilenet model and show both the outputs in the same feed. 

**Describe the solution you'd like**
Can I use the same application to take same camera input frame and then run interpreter for two models of object detection and show both the outputs. 

**Additional context**
Could you please guide to use two models.
- Loading the models
- Running two interpreters
- Showing both the outputs.

**Comment by asus4** [comment]
@nitish11  Didn't get it to work just by running the two SSD.cs at the same time?

**Comment by nitish11** [comment]
@asus4  : Yes, I want to run two SSD models that have been trained on different hypermeters. I want to show both the outputs to evaluate in run-time.

**Comment by asus4** [comment]
@nitish11 I see. I think that SSD should work with two models. Also, please refer to the example of running two models in the BlazePose scene.
